GVRf/Framework/jni/oculus/activity.cpp

```diff
@@ -0,0 +1,233 @@
+/* Copyright 2015 Samsung Electronics Co., LTD
```

Comment:
we might need to use 2016 now. and might need to update all headers to 2016 as other open source projects have done.